react/tic-tac-toe/src/utils/utils.js

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
Same, you can configure the text editor to actually do that for you 😄

Reply:
Add a linea o remove it?